### ISSUE TITLE
unban ip range

### ISSUE THREAD

**Issue by srulikuk** [fail2ban/fail2ban#2791]
Fail2Ban v0.10.2

When unban a ip range it only unbans if the ban was on the full range i.e 
`fail2ban-client set jail banip 192.168.10.48/28` and then `fail2ban-client unban 192.168.10.48/28` will unban all ip's in the range and remove all from ipset, however `fail2ban-client set jail banip 192.168.10.52` and then `fail2ban-client unban 192.168.10.48/28` does NOT remove 192.168.10.52 from ipset. 

As a workaround I had to write a little script to loop through all ip's in the range and check ipset for each one.

Would be great if this is added to fail2ban-client.

Many thanks.

**Comment by sebres** [comment]
Indeed it'd be possible with small change. I'll take a closer look or try to implement it later.

**Comment by sebres** [comment]
Implemented in 7d172faa50db8153265b7ac0e26f1034d7d6119f.
As a bonus the range can be also supplied as DNS (FQDN, hostname) to unban any IP(s) resolved by this DNS.

Example:
```bash
# unban IPs of 2 subnets and DNS in given jail:
fail2ban-client set "$jail" unbanip 192.0.2.48/28 192.0.2.96/28 fqnd.example.com
# unban IPs of 2 subnets and DNS in every jail:
fail2ban-client unban               192.0.2.48/28 192.0.2.96/28 fqnd.example.com
```

**Comment by srulikuk** [comment]
great, many thanks
